ClientManagementService/ClientManagementService.API/Controllers/ClientController.cs

```diff
@@ -1,15 +1,156 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
+using ClientManagementService.API.DTO;
+using ClientManagementService.API.DTOMapper;
+using ClientManagementService.Domain.Services;
 using Microsoft.AspNetCore.Mvc;
 
 namespace ClientManagementService.API.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api/client")]

```

Comment:
This doesn't really matter.

When it does [controller], the route is pretty much api/client. Its the same as what you have api/client.